build/configs/artik05x/artik05x_download.sh

```diff
@@ -78,17 +79,32 @@ download()
         exit 1
     fi
 
+    # check existence of romfs.img if romfs is enabled
+    if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+        pushd ${OS_DIR_PATH} > /dev/null
+        sh ../tools/fs/mkromfsimg.sh
+        if [ ! -f "${FW_DIR_PATH}/romfs.img" ]; then
+            echo "ROMFS image is not present"
+            exit 1
```

Comment:
Even if CONFIG_FS_ROMFS is enabled, romfs.img can be missed. That is not a problem. User can enable romfs without content. Let's set some romfs variable. And at flash (program) step, we can check that and decide flashing romfs.img.